apps/daffio/src/app/core/sidebar/containers/sidebar-viewport/sidebar-viewport.component.spec.ts

```diff
@@ -12,6 +12,8 @@ import * as fromSidebar from '../../reducers/index';
 import { RouterTestingModule } from '@angular/router/testing';
 import { OpenSidebar, CloseSidebar, SetSidebarState } from '../../actions/sidebar.actions';
 import { cold } from 'jasmine-marbles';
+import { DaffioGuidesViewerModule } from 'apps/daffio/src/app/docs/shared/components/guides-viewer/guides-viewer.module';
```

Comment:
This path seems evil.

Reply:
Addressed in 5de79c050489de6e9ea9890dcbd9152c51a22c04